content/blog/newsletter/2024-09-05-newsletter.it.md

```diff
@@ -0,0 +1,274 @@
+# Benvenuti alla Newsletter XMPP, è un piacere avervi di nuovo qui! Questo numero copre il mese di agosto 2024.
+
+
+# Annunci XSF
+
+La XSF [ha firmato una lettera aperta alla Commissione europea](https://xmpp.org/2024/08/the-xsf-signs-open-letter-to-the-european-commission/). Come stanno facendo molte altre organizzazioni, la [XMPP Standards Foundation](https://xmpp.org/about/xmpp-standards-foundation/) (XSF) ha deciso di firmare anche la [Lettera aperta alla Commissione europea](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter).
+
+La [XMPP Standards Foundation](https://xmpp.org/about/xmpp-standards-foundation/) sta inoltre convocando il Board XSF 2024 e il Consiglio XSF 2024. Partecipate alle decisioni dell'organizzazione della XMPP Standards Foundation e alle specifiche che pubblichiamo. Se siete interessati a candidarvi per il Board o il Council, aggiungete una pagina [wiki](https://wiki.xmpp.org/web/Main_Page) sulla vostra candidatura a [una o entrambe](https://wiki.xmpp.org/web/Board_and_Council_Elections_2024) delle seguenti sezioni fino al 3 novembre 2024, ore 00:00 UTC. Nota: i membri del Consiglio devono essere membri eletti dell'XSF; tuttavia, non c'è questa restrizione per il Board.
+
+
+# XMPP e Google Summer of Code 2024
+
+L'XSF è stato accettato come [organizzazione ospitante della GSoC nel 2024](https://wiki.xmpp.org/web/Google_Summer_of_Code_2024)! Questi progetti XMPP hanno ricevuto uno slot e hanno iniziato la loro attività di codifica:
+
+- [Monal](https://monal-im.org/)
+    - [Modern Monal onboarding e Media gallery (175 ore, medio)](https://wiki.xmpp.org/web/Gsoc2024/Monal/Media_Gallery)
+        - [Post del blog 1](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-1-xmpp-standards-foundation-da781ac95560)
+        - [Post sul blog 2](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-2-midterm-evaluations-xmpp-standards-foundation-3be8b27dc653)
+
+
+<center>
```

Comment:
@cal0pteryx that works well?

Reply:
No, this is HTML, which won't be rendered by Hugo. We use shortcodes for this on xmpp.org